warehouse/query-core/src/main/java/datawave/query/jexl/visitors/ShortCircuitBaseVisitor.java

```diff
@@ -0,0 +1,216 @@
+import org.apache.commons.jexl2.parser.ASTReturnStatement;
+import org.apache.commons.jexl2.parser.ASTSizeFunction;
+import org.apache.commons.jexl2.parser.ASTSizeMethod;
+import org.apache.commons.jexl2.parser.ASTStringLiteral;
+import org.apache.commons.jexl2.parser.ASTTernaryNode;
+import org.apache.commons.jexl2.parser.ASTTrueNode;
+import org.apache.commons.jexl2.parser.ASTUnaryMinusNode;
+import org.apache.commons.jexl2.parser.ASTWhileStatement;
+import org.apache.commons.jexl2.parser.SimpleNode;
+
+public class ShortCircuitBaseVisitor extends BaseVisitor {
+    
+    @Override
+    public Object visit(SimpleNode node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTBlock node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTAmbiguous node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTIfStatement node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTWhileStatement node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTForeachStatement node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTTernaryNode node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTBitwiseOrNode node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTBitwiseXorNode node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTBitwiseAndNode node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTAdditiveNode node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTAdditiveOperator node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTMulNode node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTDivNode node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTModNode node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTUnaryMinusNode node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTBitwiseComplNode node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTIdentifier node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTNullLiteral node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTTrueNode node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTFalseNode node, Object data) {
+        return data;
+    }
+    
+    @Override
+    public Object visit(ASTIntegerLiteral node, Object data) {
```

Comment:
The Integer/Float literals appear to be deprecated when I viewed this branch in IDE, would it still be necessary to override? Just wondering.